src/commcare_cloud/ansible/roles/commcarehq/templates/localsettings.py.j2

```diff
@@ -374,6 +374,14 @@ CELERY_FLOWER_URL = '{{ CELERY_FLOWER_URL }}'
 CELERY_TIMEZONE = '{{ localsettings.CELERY_TIMEZONE }}'
 {% endif %}
 
+{% if 'CAPTCHA_LENGTH' in localsettings and localsettings.CAPTCHA_LENGTH %}
+CAPTCHA_LENGTH = '{{ localsettings.CAPTCHA_LENGTH }}'
```

Comment:
Is this meant to be a string or a number? A number I think in which case you should remove the quotes:
```suggestion
CAPTCHA_LENGTH = {{ localsettings.CAPTCHA_LENGTH }}
```

Reply:
sorry yes, you are right. When i deployed this on icds-staging for testing, i did remove them but forgot to change in the changes i had locally.